js/audio/shared/WaveformRecordControls.svelte

```diff
@@ -195,6 +195,10 @@
 		border: 1px solid var(--block-border-color);
 	}
 
+	.duration-button {
```

Comment:
Just a styling tweak for consistency:

<img width="390" alt="image" src="https://github.com/user-attachments/assets/07a4cb3e-b5e6-48a4-8dcc-006cd4204a01" />